packages/contracts/contracts/chugsplash/ChugSplashDeployer.sol

```diff
@@ -0,0 +1,171 @@
+// SPDX-License-Identifier: MIT
+// @unsupported: ovm
+pragma solidity >0.5.0 <0.8.0;
+
+import { ChugSplashProxy } from "./ChugSplashProxy.sol";
+
+/**
+ * @title ChugSplashDeployer
+ */
+contract ChugSplashDeployer {
```

Comment:
Consider tradeoff of shutting down the system being encoded into the contract vs in the bundle